blocks/components/editable/tinymce.js

```diff
@@ -3,7 +3,11 @@ export default class TinyMCE extends wp.element.Component {
 		this.initialize();
 	}
 
-	shouldComponentUpdate() {
+	shouldComponentUpdate( nextProps ) {
+		if ( this.editorNode.getAttribute( 'data-is-empty' ) !== nextProps.isEmpty ) {
```

Comment:
>Are we guaranteed it has the same bounds? You could have multiple Editables in a row?

My suggestion would be to apply the placeholder at `Editable`, not the block, so it shouldn't be an issue to have consecutive Editables.